### PR TITLE
Changelog v1.7.7.0

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Update Changelog v1.7.7.0
| Type?         | new feature
| Category?     | PM
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | ~
| How to test?  | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22064)

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Review by matks — DISMISSED** [review]
I block this PR to make sure nobody merges it by mistake

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -24,8 +24,46 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
...
+- Core:
+  - Bug fix:
+    - #21975:  Correctly substring fields before update, remove duplicates and add missing sql queries (by @PierreRambaud)
+    - #GHSA-frf2-c9q3-qg9m:  Improper Access Control with submitReorder function (by @PierreRambaud)
+- Installer:
+  - Improvement:
+    - #21957:  Update latest native modules (by @jolelievre)
+  - Bug fix:
+    - #21953:  Fix "column count doesn't match value count" error during upgrade (by @okom3pom)
+    - #21158:  Bump version to 1.7.6.8 in the installer (by @PierreRambaud)
```
Comment (eternoendless) [inline docs/CHANGELOG.txt]: You can remove this one
```suggestion
```

**Review by Progi1984 — DISMISSED** [review]

**Review by eternoendless — CHANGES_REQUESTED** [review]
LGTM, but do not merge yet!

**Review by eternoendless — DISMISSED** [review]
LGTM, wait for validation

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**